plugins/woocommerce/client/legacy/css/admin.scss

```diff
@@ -2698,12 +2698,13 @@ ul.wc_coupon_list_block {
 			color: #ccc;
 			display: inline-block;
 			cursor: pointer;
-			padding: 0 0 0.5em;
-			margin: 0 0 0 12px;
+			padding: 0;
+			margin: 0 0 0.5em 12px;
```

Comment:
Can you please explain why did you use `em` and `px` in the same context?

Reply:
@naman03malhotra Before this PR, the icons were aligned with the text by the padding-bottom value (`padding: 0 0 0.5em;`). But this is what the focus ring looked like:
![image](https://github.com/woocommerce/woocommerce/assets/9420947/033b1d5c-1769-4125-89b5-5ea97e4ecb1d)

To remove the extra space below the icon, I decided to remove the bottom spacing from the `padding` and add it to the `margin` attribute: `margin: 0 0 0.5em 12px`;. 
![image](https://github.com/woocommerce/woocommerce/assets/9420947/7c2f9d2e-8f74-4196-9527-b56352cd4527)

To keep it consistent as it was before the change, I kept the `0.5em` value.

Please let me know your thoughts.

Reply:
Good practice would be to follow a consistent metric throughout, can you check if we can use px in both places?

Reply:
@naman03malhotra I fixed it under this commit: [Replace em unit with px](https://github.com/woocommerce/woocommerce/pull/48238/commits/09ad03f754d264536ef9950b571e8d632354adb5)